package.json

```diff
@@ -7,6 +7,7 @@
     "build": "vite build",
     "preview": "vite preview",
     "docs": "jsdoc --readme ./docs/README.md -c jsdoc.config.json",
+    "docs:clear": "cd docs && mv README.md ../temp.md && rm -rf * && cd ../ && mv temp.md docs/README.md",
```

Comment:
This is good, but it won't work on Windows. Can you refactor this into a JS script instead? That should make it platform agnostic.

Reply:
Sure! I'll refactor this.